### DISCUSSION TITLE
when is HW1 due?

### DISCUSSION THREAD

**Discussion by KatLeigh11** [course-fish521-2023/assistance#2] [Q&A] [answered]
Hi all,

Anyone know the due date for HW1? I missed that bit of info. Thanks!
Best
Kat

**Comment by sr320** [comment, marked as answer]
Great Question - Due dates for all HW are available at 
https://sr320.github.io/course-fish521-2023/homework.html

(essentially the next Monday) 

And technically 1:30 .... https://sr320.github.io/course-fish521-2023/index.html#Evaluation
